source/defclass-star.lisp

```diff
@@ -522,3 +522,58 @@ See `define-class' for more details."
      `(define-condition ,name ,supers
         ,processed-slots
         ,@clean-options))))
+
+(defmacro make-instance* (class &rest rest)
```

Comment:
Why not call it `make`? :)

Reply:
Could work too. But then, most of our star-macros are just variations of default names with stars appended to the end (see the top post for the discussion of names consistency). And `make` is quite generic too, which is not cool.

But I can see the use of it, with e.g. Serapeum having their convenience macro `make`.